automatic/brave/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,16 @@
+﻿$registry = Get-UninstallRegistryKey -SoftwareName "brave*"
```

Comment:
I thought the automatic uninstaller would be able to uninstall the software.
Has this changed and is no longer picked up automatically?

This uninstall script is also very error prone, and will fail the uninstallation if the program had already been uninstalled.

See https://github.com/chocolatey-community/chocolatey-packages/blob/master/automatic/7zip.install/tools/chocolateyUninstall.ps1 as an example on how the uninstall script should be implemented.